GestureHandler.js

```diff
@@ -368,8 +338,9 @@ function createHandler(
           })
         );
       }
+
       return React.cloneElement(
-        child,
+        GestureHandlerModule.render(handlerName, this.props),
```

Comment:
on web we seem to have handler specific components that we render here. So in my option it does not make sense to render and immediately clone it. So maybe lets only use clone on mobile where we want to clone the underlying child component?

Another aspect of that is it appears to me like we inadvertently try to reuse logic of the component returned from `createHandler` while it seem the whole logic there is only so that we call `GestureHandlerModule` methods which are no-ops on web. So in my opinion we should diverge this implementation at an earlier stage.

Reply:
I just restructured it once again and introduced a `GestureHandlerRenderer.js` which should improve this. e.g., on web we do not need to render the debug views explicitly.